integration-tests/src/server.js

```diff
@@ -81,6 +82,7 @@ _.forEach(registerTestEndpoint, [
   testRegisterUser,
   testRunNotebook,
   testRunWorkflow,
+  testRequestAccess,
```

Comment:
I'm actually not sure if I need to update this file 🤔

Reply:
You can remove this.
This file starts a server that was intended for perf testing. QA could call the tests in rapid succession for testing. I don't think it has been used in a few years.